commands/deploy.go

```diff
@@ -42,16 +42,12 @@ type UploadResponse struct {
 
 // Run deploys an app to Section's edge
 func (c *DeployCmd) Run() (err error) {
-	if c.Debug {
-		fmt.Println("Server URL:", c.ServerURL.String())
-	}
-
 	ignores := []string{".lint/", ".git/"}
 	files, err := BuildFilelist(c.Directory, ignores)
 	if c.Debug {
-		fmt.Println("Archiving files:")
+		fmt.Println("[debug] Archiving files:")
```

Comment:
Consider the [hashicorp logging package](https://github.com/hashicorp/logutils) that we use in many other Go projects. It can filter the log output just by the presence of prefixes like `[debug]` in the written string without needing to litter code with `if loglevel {}` blocks.

Reply:
@jstangroome yep good call. 

Now our logging is becoming more verbose, we're gonna better sensible defaults for logging.